flow-nodes/api-builder-plugin-md5/src/flow-nodes.yml

```diff
@@ -5,23 +5,29 @@ flow-nodes:
     description: MD5 message digest
     category: general
     methods:
-      digest:
+      digest: # This same key is used in `actions.js`
+        name: MD5 Digest
         description: Creates an MD5 digest and returns as hex.
         parameters:
           data:
+            name: Data
             description: The data to digest.
             required: true
             initialType: string
-        outputs:
-          next:
-            name: Next
-            description: Successfully created digest.
-            context: $.md5
-            schema:
-              type: string
-          error:
-            name: Error
-            description: Encountered an error.
-            context: $.error
             schema:
+              title: Data
```

Comment:
No point passing title and name and the same time since they just overwrite each other. Name is preferred.

Reply:
This is fixed now.